Helper/Elasticsearch/ClientRequestRuntime.php

```diff
@@ -103,12 +105,10 @@ public function get(string $input)
                 ['term' => ['_contenttype' => $emsLink->getContentType()]],
             ];
         }
-
         $result = $this->manager->getDefault()->searchArgs(['body' => $body]);
 
-
         if (1 === $result['hits']['total']) {
-            return $result['hits']['hits'][0]['_source'];
+            return new Document($emsLink->getContentType(), $emsLink->getOuuid(), $result['hits']['hits'][0]['_source']);
```

Comment:
Document should receive $result['hits']['hits'] and derive content type, ouuid and other info in the constructor.
We could also pass the emsLink object to the constructor, so that the Document can do verifications? (match between info in emsLink and the returned $result. 

Also the ->getContentType and ->getOuuid calls should be done inside the constructor, to prevent that we do them manually each time a new Document is created throughout the codebase.